src/main/java/hasebo/scrumpoker/model/Voting.java

```diff
@@ -0,0 +1,29 @@
+package hasebo.scrumpoker.model;
+
+import jakarta.persistence.*;
+import lombok.NoArgsConstructor;
+import java.util.List;
+
+@Entity
+@Table(name="voting")
+@NoArgsConstructor
+public class Voting {
```

Comment:
czy to potrzebne w ogóle?

Reply:
Potrzebne, w pokoju będzie wiele głosowań, jedno będzie mogło być aktywne, reszta nieaktywne=historyczne po zakończeniu głosowania.